Example/Examples/CollectionViewExample/HorizontalListFactory.swift

```diff
@@ -16,18 +16,21 @@ struct HorizontalListFactory {
 
     func make() -> [CollectionViewSection] {
         let section = StaticCollectionSection(
-            cellBuilders: cellBuilders()
+            cellBuilders: cellBuilders(),
+            spacingBetweenItems: (5.0, 5.0),
+            sectionInsets: .zero
         )
         let section2 = StaticCollectionSection(
-            cellBuilders: cellBuilders()
+            cellBuilders: cellBuilders(),
+            spacingBetweenItems: (5.0, 5.0),
+            sectionInsets: UIEdgeInsets(top: 8.0, left: 8.0, bottom: 8.0, right: 8.0)
```

Comment:
This crashes the `HorizontalList` example. It shouldn't have vertical (top and bottom) insets by now. I have to figure out why this happens tho